api/src/main/java/org/pmiops/workbench/institution/InstitutionServiceImpl.java

```diff
@@ -357,6 +357,14 @@ public boolean validateOperationalUser(DbInstitution institution) {
         .map(dbi -> institutionMapper.dbToModel(dbi, this));
   }
 
+  @Override
+  public List<DbUser> getAffiliatedUsers(String shortName) {
```

Comment:
The shortName (which is a unique key) is exposed in the API and the DB ID is not, so that's what is generally used as the accessor in this Service and Controller.

OTOH: we could change to a pattern where the Controller does an ID lookup on the shortName before calling the Service with an ID.  I don't have a strong preference.

Reply:
If I'm understanding this correctly, then the shortName can be updated by the admin. Seems like this is the source of a bug... retrieving from the db with a key that isn't immutable.

Reply:
Yes, admins can change an inst shortName.  If you are concerned that this can lead to bugs (makes sense) we can remove this ability.  It's not important to keep it. 

Or are are you referring to a specific bug?

Reply:
So in the case that shortName has mutated, any calls in progress with the old shortName with throw a NotFoundException. If the db call used an immutable key this would not happen. If this is only used in the admin panel, then it's less concerning, but if any API calls are dependent on this mutable key, it's a bit concerning.